### ISSUE TITLE
Error running tests

### ISSUE THREAD

**Issue by tim-coder** [danny0838/webscrapbook#400]
Hi, and thanks for the excellent browser extension.

When I run the test suite, I get the following error message:

```
isUtc = false
[‣](moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/test.html?grep=lib%2Fstrftime%5C.js%20Strftime%5C.format%20isUtc%20%3D%20false)

AssertionError: expected '19' to equal '20'AssertionError@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/lib/chai.js:9227:13
[3]</module.exports/Assertion.prototype.assert@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/lib/chai.js:253:13
assertEqual@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/lib/chai.js:1438:12
methodWrapper@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/lib/chai.js:7910:25
[6]</module.exports/assert.strictEqual@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/lib/chai.js:4445:58
@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/test_lib_strftime.js:43:14
@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/test.js:411:9
async*@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/test.js:412:3
@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/test.js:20:12
@moz-extension://2313bf9d-3ddd-45a6-a36c-c4a769be2fb4/test.js:26:2
```

My computer is running Ubuntu 22.04.1
Browser: Firefox 133.0

Please advise.

Thx!

**Comment by danny0838** [comment]
How did you run the test? Where and for which test does the error show up? Can you provide a screenshot to illustrate that?

**Comment by tim-coder** [comment]
Actually, there are three errors. See attached.

[WebScrapBook Test Suite.pdf](https://github.com/user-attachments/files/18069681/WebScrapBook.Test.Suite.pdf)

**Comment by tim-coder** [comment]
BTW, I ran the tests using the WebScrapBook Test Suite, menu option:  Run automated tests

**Comment by danny0838** [comment]
For the `Strftime.format` test: please check if [the revised version](https://github.com/danny0838/webscrapbook/tree/fix-400) fixes the error.

For the alternative stylesheets tests: please check whether re-running the tests still fail.

**Comment by tim-coder** [comment]
The revised version gives me errors also. See attached.

[WebScrapBook Test Suite-2.pdf](https://github.com/user-attachments/files/18089400/WebScrapBook.Test.Suite-2.pdf)

**Comment by danny0838** [comment]
I don't think the result you provided should happen. Have you done the update and reloaded all extensions?

**Comment by tim-coder** [comment]
My bad. I forgot to update the test suite extension.

When both extensions are used from revised version, the test suite does not run. Attached is the console for the WebScrapBook Test Suite.

![Screenshot from 2024-12-11 06-43-09](https://github.com/user-attachments/assets/f3c5aa9b-8412-47e0-87a0-72a8e307b871)

**Comment by danny0838** [comment]
Have you restarted the test server?

**Comment by tim-coder** [comment]
Yes

**Comment by danny0838** [comment]
The error should only happen when the tests isn't run correctly.

Please make sure to shutdown all existing test server windows and then restart the test server of the new version.

**Comment by tim-coder** [comment]
I closed the browser and stopped the server. Then I started the server, started the browser, installed WebScrapBook, installed WebScrapBook TestSuite (both using Temporary Add-ons), and got the same result.

I'll try it with my Chrome browser to see what results I get.

**Comment by tim-coder** [comment]
With Chrome (Version 131.0.6778.108 (Official Build) (64-bit)), the test suite runs, but I get multiple errors.

![Screenshot from 2024-12-11 08-14-47](https://github.com/user-attachments/assets/8d4bbf37-322c-4bd8-874c-954a2c6f8bd4)
![Screenshot from 2024-12-11 08-14-25](https://github.com/user-attachments/assets/4d932933-3ee6-4f18-a09a-b9ffd96f28af)
![Screenshot from 2024-12-11 08-14-03](https://github.com/user-attachments/assets/fcbd7da0-8f3b-4320-af9f-06d59e04d505)
![Screenshot from 2024-12-11 08-13-47](https://github.com/user-attachments/assets/bb51fc08-9e0c-474b-892a-66ef5ee52ff0)
![Screenshot from 2024-12-11 08-12-22](https://github.com/user-attachments/assets/8c29fbc3-05cf-47d3-b136-5cb3692eb9b9)
![Screenshot from 2024-12-11 08-11-36](https://github.com/user-attachments/assets/2a58496f-e5b7-45d9-9bcd-789e8ba4d3f5)

**Comment by danny0838** [comment]
For the `Strftime.format` test: again, I suspect you have not run the test using the requested revised version correctly.

For other tests: you probably forget to config the server URL or server port is used by another app.

**Comment by tim-coder** [comment]
You're right about the server. When I (re)installed the extensions, I forgot to set the backend server address in WebScrapBook.

After updating the server address, Firefox behaves the same - no tests are run.

With Chrome, I get the `Strftime.format` error. Later, the testing stops/hangs at 83% complete.

Is there anything else I need to configure to run the test suite?

![Screenshot from 2024-12-11 09-10-22](https://github.com/user-attachments/assets/fc625bc3-ccab-447b-80f7-dde30ec3064f)

**Comment by danny0838** [comment]
Do you make sure that everything is done with the provided revised version? I don't think the error should appear with that version...

For the hang issue, it's probably because you have paused the server (such as clicking in it's dialog)

**Comment by tim-coder** [comment]
OK, I think I've gotten my user errors straightened out. I had my directories mixed up.

Firefox and Chrome behave the same with the latest update - getting only the one error in the `Strftime.format` test.

Is there anything else I should try?

Firefox
![Screenshot from 2024-12-11 21-31-22](https://github.com/user-attachments/assets/38540562-343b-495a-ab9b-2b9d7bd7fff1)


Chrome
![Screenshot from 2024-12-11 21-30-18](https://github.com/user-attachments/assets/cf62e31a-4610-4cbe-9ca5-05d7afdf41d7)

**Comment by danny0838** [comment]
I don't think you're testing the correct version. Please be sure to get a clean copy of [the given commit](https://github.com/danny0838/webscrapbook/tree/b6757648c7740599aba2a7ea14102be8383b6ae6) and perform a test after cleaning everything.

**Comment by tim-coder** [comment]
I got the test to work correctly checking out  the commit from your previous message. I must have made a mistake when I used your fix-400. It's all good now.

Thanks for your patience.
